exporter.go

```diff
@@ -343,10 +371,22 @@ func lineToEvents(line string) Events {
 		log.Debugln("Bad line from StatsD:", line)
 		return events
 	}
-	metric := elements[0]
+
+	labels := map[string]string{}
+	metric := parseNameAndTags(elements[0], labels)
+
 	var samples []string
 	if strings.Contains(elements[1], "|#") {
-		// using datadog extensions, disable multi-metrics
+		// using DogStatsD tags
+
+		// don't allow mixed tagging styles
+		if len(labels) > 0 {
+			sampleErrors.WithLabelValues("malformed_line").Inc()
```

Comment:
Let's count it separate from a generally malformed line:

```suggestion
			sampleErrors.WithLabelValues("mixed_tagging_styles").Inc()
```

Reply:
Works for me.